core/src/main/java/org/apache/calcite/tools/RelBuilder.java

```diff
@@ -2495,12 +2504,6 @@ public RelBuilder permute(Mapping mapping) {
     return project(exprList);
   }
 
-  public RelBuilder aggregate(GroupKey groupKey,
-      List<AggregateCall> aggregateCalls) {
-    return aggregate(groupKey,
-        Lists.transform(aggregateCalls, AggCallImpl2::new));
-  }
-
   /** Creates a {@link Match}. */
```

Comment:
Is this change expected? It seems not necessary.

Reply:
Not that necessary, just move the code where it should belong.